python/cudf/cudf/tests/test_json.py

```diff
@@ -725,7 +723,7 @@ def test_json_types_data():
 def test_json_quoted_values_with_schema(col_type, json_str):
     experimental_df = cudf.read_json(
         StringIO(json_str),
-        engine="cudf_experimental",
+        engine="cudf",
```

Comment:
The Line 732 engine should be different.

similarly at some other places too.

Reply:
Done